source/_integrations/nice_go.markdown

```diff
@@ -32,3 +34,7 @@ Garage doors linked to your Nice G.O. account will appear as covers.
 ## Light
 
 Lights on your garage door will appear as lights.
+
+## Switch
+
+A switch for turning vacation mode on and off will be made available for each device.
```

Comment:
In the PR description you give a bit more detail. Would it make sense to have that here as well?